packages/front-end/components/SavedGroups/IdListItemInput.tsx

```diff
@@ -51,6 +52,10 @@ export const IdListItemInput: FC<{
   );
   const [fileName, setFileName] = useState("");
   const [fileErrorMessage, setFileErrorMessage] = useState("");
+  const orgBypassSmallListSizeLimit = !useFeatureValue(
+    "apply-saved-group-id-list-size-limit",
+    false
```

Comment:
I have a default-false here so if the instance doesn't get the GB features payload it won't block users. Then I have the dogfooding feature set up with default-true + opt-out rules for specific orgs. That sound right @jdorn?